.eslintignore

```diff
@@ -2,5 +2,17 @@ node_modules
 build
 dist
 coverage
-!.commitlintrc.js
+.commitlintrc.js
+.eslintrc.js
+.prettierrc.js
 **/*.d.ts
+packages/core/output-target
+packages/core/**/stories
+packages/core/loader
```

Comment:
Question: Why ignore those packages too?

Reply:
`output-target` and `loader` is autogenerated files so is ok

`stories` is a problem because sometimes we need to use specific things to run web components with storybook

What you think?

Reply:
Sorry, I wasn't clear. 😅  I'm thinking about packages/core/react, packages/core/vue, packages/react and packages/vue

Reply:
everyone are also auto generated from core, so they cannot pass lint rules

but I'm not sure if we should keep `/stories`, what do you think?

Reply:
thanks for the explanation, man! 

> but I'm not sure if we should keep /stories, what do you think?

Yeah... Guess we could test it and then get some feedback.

Reply:
Uhmmm, for now I we keep it outside of ignore, if we have some problems we could add again [bc3fab3](https://github.com/juntossomosmais/atomium/pull/269/commits/bc3fab332e843e9cec5baede5098f3bf7cacee96)